app/components/works/dates_component.html.erb

```diff
@@ -129,13 +150,20 @@
                 { prefix: prefix, field_name: 'created_range(6i)', prompt: 'day' },
                 data: { date_validation_target: 'day', 
                   date_range_target: 'endDay',
+                  date_clear_target: 'day',
                   date_approximate_target: 'day',
                   action: 'date-range#change date-approximate#change' },
                 id: "#{prefix}_created_range_end_day", class: "form-control" %>
           </div>
+          <%= button_tag type: 'button', class: 'btn btn-sm ms-3', aria: { label: "Clear" },
```

Comment:
Can this label be more descriptive than "Clear"? "Clear created date range" perhaps?

Reply:
Yes. Done.